.github/workflows/container-upload.yml

```diff
@@ -0,0 +1,49 @@
+name: Container Upload Workflow
+
+on:
```

Comment:
Where is this workflow used?

Reply:
Good catch, this is a left over when I tried to make this work with workflow_call, but that was rather iffy.